utils/hct/hctdb.py

```diff
@@ -2730,6 +2743,8 @@ def build_semantics(self):
             Barycentrics,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NotPacked _61,NA,NA,NA,NA,NA,NA
             ShadingRate,NA,SV _64,NA,NA,SV _64,SV _64,NA,NA,SV _64,SV _64,SV _64,NA,SV _64,SV _64,NA,NA,NA,NA,SV,NA
             CullPrimitive,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NotInSig,NA,NA,NA,NA,NotPacked,NA
+            BaseVertexLocation,NotInSig _68,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA
```

Comment:
Why are these being implemented as semantics if they are not in the signature? Wouldn't it make more sense to have these be Intrinsics in that case?

Reply:
This is based on https://github.com/microsoft/DirectXShaderCompiler/issues/2907

We could discuss if intrinsic like getBaseVertexLocation is better for this.
That will be easier than semantic for implementation :)

Reply:
Since this is a vertex shader system generated value, I think using the current approach would keep things consistent with other special values in that shader stage.

Reply:
Also, I'm not sure how a front-end intrinsic is easier to implement than an added item in an existing pattern, where the majority of the code changes are generated (and has now been implemented in this PR).  If you mean implementation in the back-end, it is an intrinsic in DXIL, so there's no difference there.

Reply:
I mean intrinsic will have the type set correctly, no need to check to make sure SV_BaseVertexLocation is applied on int.